lib/inspec/schema/schema.rb

```diff
@@ -0,0 +1,64 @@
+require "json"
+require "lib/inspec/schema/primitives"
+require "lib/inspec/schema/exec_json"
+require "lib/inspec/schema/exec_json_min"
+require "lib/inspec/schema/profile_json"
+
+module Inspec
+  module OutputSchema
```

Comment:
and should be namespaced to match.

Reply:
If I'm being fully honest I'd never written ruby until ~2 weeks ago, so still not entirely clear on some best-practice stuff. 

To be clear, are you saying it should be
`
namespace OutputSchema
    module OutputSchema
`

Apologies if I'm being obtuse here.

Reply:
```
module Inspec
  module Schema
    module OutputSchema
    end
  end
end
```

Reply:
Resolved as of https://github.com/inspec/inspec/pull/4330/commits/e7cb80d927b114bcbb0a3627dba84805f2be3300